frontend/services/viewer/viewer.tsx

```diff
@@ -895,8 +895,7 @@ export class ViewerService {
 	 */
 
 	public async getObjectsStatus({ teamspace, model } = { teamspace: '', model: '' }) {
-		await this.isViewerReady();
-		return this.getUnityObjectsStatus(teamspace, model);
+		return this.isInitialised ? await asyncTimeout(1000, this.getUnityObjectsStatus, teamspace, model) : null;
```

Comment:
why do we want to wait then return? what if it's still not initialised?

Reply:
If I remember correctly the  problem is that when you are in the kanban and you havent entered a model isViewerReady never finishes so this function gets unresponsive and you cant create issues without entering first a model